# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/silo/debug.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- silo-fcf8ffb4fba3ea72353c111ab7dc210eaae89223/second/file.c.vanilla	2021-02-10 17:07:43.204865718 +0100
+++ silo-fcf8ffb4fba3ea72353c111ab7dc210eaae89223/second/file.c	2021-02-10 17:09:37.740871626 +0100
@@ -195,6 +195,7 @@
             }
             if ((char *)filebuffer + (block_cnt + ((*blocknr) ? (blockcnt - last_blockcnt - 1) : 0)) * bs > filelimit) {
                 silo_fatal("Image too large to fit in destination");
+                printf("\nImage too large to fit in destination: %d\n", (long)filelimit - (long)filebuffer);
                 return BLOCK_ABORT;
             }
             if (block_cnt > 0 && io_channel_read_blk (fs->io, block_no, block_cnt, filebuffer))
